packages/peregrine/lib/Router/magentoRouteHandler.js

```diff
@@ -35,9 +35,10 @@ export default class MagentoRouteHandler extends Component {
                 'Current environment does not support CacheStorage at window.caches.'
             );
         }
-        const myCache = await window.caches.open(
-            `workbox-runtime-${location.origin}/`
-        );
+        /**
+         * TODO - Should take cache name from defaults
```

Comment:
@jimbo where do you think we should have this constant stored? It is a configurable variable, we can not hardcode it in venia/peregrine.

I propose the `.env` file. It is a common file that both webpack bundlers (Client and SW) have access to. Webpack can inject them into the respective code bases as process variables.